### PR TITLE
Update get-remediation-all-activities.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document with odata filter queries

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5924

File: microsoft-365/security/defender-endpoint/get-remediation-all-activities.md [hunk 1]
```diff
@@ -43,6 +43,13 @@ Returns information about all remediation activities.
 
 **URL:** GET: /api/remediationTasks
 
+<br>Supports [OData V4 queries](https://www.odata.org/documentation/).
+<br>OData supported operators:
+<br>```$filter``` on:  ```createdon``` and ```status``` properties.
+<br>```$top``` with max value of 10,000 
+<br>```$skip```
+<br>See examples at [OData queries with Microsoft Defender for Endpoint](exposed-apis-odata-samples.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-remediation-all-activities.md]:
```suggestion
<br>Supports [OData V4 queries](https://www.odata.org/documentation/).
<br>OData supported operators:
<br>```$filter``` on:  ```createdon``` and ```status``` properties.
<br>```$top``` with max value of 10,000.
<br>```$skip```.
<br>See examples at [OData queries with Microsoft Defender for Endpoint](exposed-apis-odata-samples.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@siosulli Gentle reminder

**Pull request merged**